Code/ForceField/Wrap/ForceField.cpp

```diff
@@ -194,8 +194,11 @@ python::tuple PyForceField::minimizeTrajectory(unsigned int snapshotFreq,
   int resInt = this->field->minimize(snapshotFreq, &snapshotVect, maxIts,
                                      forceTol, energyTol);
   python::list l;
+  boost::python::manage_new_object::apply<RDKit::Snapshot *>::type converter;
   for (const auto &it : snapshotVect) {
-    l.append(new RDKit::Snapshot(it));
+    // transfer ownership to python
+    python::handle<> handle(converter(new RDKit::Snapshot(it)));
+    l.append(handle);
```

Comment:
This is a really useful pattern that I hadn't seen before

Reply:
I hadn't either. I saw the leak reported by asan, and when I looked at the code, I noticed that we didn't transfer ownership of the object to Python, so I googled a bit and found that pattern.